V1Documentation.md

```diff
@@ -1,53 +1,52 @@
-# Raygun4js
+# Raygun4js V1 Documentation
 
-[Raygun.io][raygunio] provider for client-side JavaScript
+The following documentation is for Raygun4js V1 only. Raygun4js V1 should only be used by legacy customers or for applications that block scripts.
 
-[raygunio]: https://raygun.io
+Go to [V2 documentation][v2docs].
+
+[v2docs]: https://github.com/MindscapeHQ/raygun4js/blob/master/README.md
 
 ## Getting Started
 
 **Step 1**
 
-Add this snippet to your markup, before the closing `</head>` tag:
+Download the [production version][min] or the [development version][max]. 
 
-```javascript
-<script type="text/javascript">
-    !function(a,b,c,d,e,f,g,h){a.RaygunObject=e,a[e]=a[e]||function(){
-    (a[e].o=a[e].o||[]).push(arguments)},f=b.createElement(c),g=b.getElementsByTagName(c)[0],
-    f.async=1,f.src=d,g.parentNode.insertBefore(f,g),h=a.onerror,a.onerror=function(b,c,d,f,g){
-    h&&h(b,c,d,f,g),g||(g=new Error(b)),a[e].q=a[e].q||[],a[e].q.push({
-    e:g})}}(window,document,"script","//cdn.raygun.io/raygun4js/raygun.min.js","rg4js");
-</script>
-```
+You can also download a version without the jQuery hooks if you are not using jQuery or you wish to provide your own hooks. Get this as a [production version][min.vanilla] or [development version][max.vanilla].
+
+[min]: https://raw.github.com/MindscapeHQ/raygun4js/master/dist/raygun.min.js
+[max]: https://raw.github.com/MindscapeHQ/raygun4js/master/dist/raygun.js
+[min.vanilla]: https://raw.github.com/MindscapeHQ/raygun4js/master/dist/raygun.vanilla.min.js
+[max.vanilla]: https://raw.github.com/MindscapeHQ/raygun4js/master/dist/raygun.vanilla.js
 
-This will fetch the raygun4js script from our CDN asynchronously, so it doesn't block other scripts from being loaded. It will also catch errors that are thrown while the page is loading, and send them when the script is ready.
+Then include the script like any other library:
+
+```
+<script src="path/to/raygun.min.js" type="text/javascript"></script>
+```
 
 **Step 2**
 
 Add the following lines to your JS site code and paste in your API key (from your Raygun dashboard), to set up the provider to automatically send errors to your Raygun:
 
 ```javascript
 <script type="text/javascript">
-    rg4js('apiKey', 'paste_your_api_key_here');
-    rg4js('enableCrashReporting', true);
+  Raygun.init('yourApiKey', { // Other setup options here});
```

Comment:
That comment won't produce a syntactically valid line with the brackets after it, suggest there's a note with a link to this section: https://github.com/MindscapeHQ/raygun4js/pull/230/files#diff-10512c1bf63d296c6eb4e2faf3de6a24R166